front/app/components/EsriMap/index.tsx

```diff
@@ -103,6 +106,47 @@ const EsriMap = ({
 
   const mapRefAvailable = !!mapRef.current;
 
+  // For Accessibility: add (aria-labels) to the expand buttons for layer list and legend
+  const { formatMessage } = useIntl();
+  useEffect(() => {
+    if (!mapView) return;
+
+    const labelExpandButtons = () => {
+      document
```

Comment:
I think it could be nice to move as much of this code into a utils.ts file instead, so we reduce the amount of code in the EsriMap index file?